high_school/views.py

```diff
@@ -199,8 +207,12 @@ def get_context_data(self, *, object_list=None, **kwargs):
                     selected_school = HighSchool.objects.filter(dbn=self.dbn)
```

Comment:
use get() to obtain a single object when filtering using primary key

Reply:
So, I tried using get instead of using filter, but the test cases fail. It fails to retrieve the object created when using get query "high_school matching query does not exist". I tried debugging it, but I'm not able to ...if you could help me out with this.